api/trip/views.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import generics
+from .serializers import TripSerializer, ActivitySerializer
+from .models import Trip, Activity
+from django.views.generic import ListView, DetailView
+from django.views.generic.edit import CreateView, DeleteView, UpdateView
+
+
+class TripList(generics.ListCreateAPIView):
```

Comment:
Can you work with the Auth team and add permissions here? We only want admins to be able to create trips and activities